src/reducers/index.js

```diff
@@ -1,12 +1,63 @@
 import { enableBatching } from 'utils/redux';
-import systemReducer from 'reducers/system';
-import feedsReducer from 'reducers/feeds';
-import cdpsReducer from 'reducers/cdps';
-
-const rootReducer = ({ system, feeds, cdps }, action) => ({
-  system: systemReducer(system, action),
-  feeds: feedsReducer(feeds, action),
-  cdps: cdpsReducer(cdps, action)
-});
+import produce from 'immer';
+import setWith from 'lodash/setWith';
+import * as math from '@makerdao/dai-plugin-mcd/dist/math';
+import ilkList from '../references/ilkList';
+import { SPOT, LIQUIDATION_RATIO, FEED_VALUE_USD } from './feeds';
+import each from 'lodash/each';
+
+const initialState = { system: {}, urns: {}, ilks: {} };
+
+const rootReducer = produce((draft, action) => {
```

Comment:
i've merged the three reducers into one, because recalculating prices for an ilk depends upon both ilk values and a system value (`par`).